lib/model/bloc/profile/ProfileListItemViewModelTransformer.dart

```diff
@@ -10,10 +14,13 @@ class SwitchProfileListItemViewModelTransformer
   SwitchProfileListItemViewModelTransformer(this._switch, this._currentProfile);
   @override
   SwitchProfileListItemViewModel transform({ProfileEntity from}) {
+    final profileViewModel = ProfileViewModel(loadingStatus: LoadingStatus.SUCCESS,initials: PersonName(from.name).initials,image: from.avatar);
```

Comment:
Try to format this adding coma just to the end of inputs. This makes code more readable without need of horizontal scroll 😄

Reply:
missing the comma, which stops the auto format working.

Reply:
sorted 👍